transport.rb

```diff
@@ -0,0 +1,18 @@
+class Transport
```

Comment:
Here we could define basic method of transport availability and toggling availability status and use it then in delivery service

Reply:
Method added